lib/puppet/type/cron.rb

```diff
@@ -18,29 +18,29 @@
 
     Example:
 
-        cron { logrotate:
-          command => "/usr/sbin/logrotate",
-          user    => root,
-          hour    => 2,
-          minute  => 0
+        cron { 'logrotate':
+          command => '/usr/sbin/logrotate',
+          user    => 'root',
+          hour    => '2',
+          minute  => '0',
```

Comment:
Since we're not writing any of these in octal or hex, I prefer all unquoted, with quotes only around the non-numeric strings like `'*/10'`.